tests/php/Http/Controllers/Api/VolumeControllerTest.php

```diff
@@ -211,4 +224,379 @@ public function testUpdateRedirect()
+            'media_type_id' => MediaType::videoId()
+        ])->fresh(); // Use fresh() to load even the null fields.
+        // The target project.
+        $project = ProjectTest::create();
+
+        $oldVideo = VideoTest::create(['volume_id' => $volume->id])->fresh();
+        $oldAnnotation = VideoAnnotationTest::create(['video_id' => $oldVideo->id]);
+
+        $this->beAdmin();
+        $project->addUserId($this->admin()->id, Role::adminId());
+
+        $response = $this->post("/api/v1/volumes/{$volume->id}/clone-to/{$project->id}");
+        $response->assertStatus(302);
+        $copy = $response->getSession()->get('copy');
+        $newVideo = $copy->videos()->first();
+        $newAnnotation = $newVideo->annotations()->first();
+
+        $this->assertNotNull($newAnnotation);
+        $this->assertNotEquals($newAnnotation->id, $oldAnnotation->id);
+        $this->assertEquals($newAnnotation->updated_at, $oldAnnotation->updated_at);
+        $this->assertEquals($newAnnotation->created_at, $oldAnnotation->created_at);
+        $this->assertNotEquals($newAnnotation->video_id, $oldAnnotation->video_id);
+        $this->assertEquals($newAnnotation->video_id, $newVideo->id);
+
+        $ignore = ['id', 'video_id'];
+        $this->assertEquals(
+            $oldAnnotation->makeHidden($ignore)->toArray(),
+            $newAnnotation->makeHidden($ignore)->toArray()
+        );
+
+    }
+
+    public function testCloneVolumeImageAnnotationLabels()
+    {
+        $volume = $this->volume([
+            'created_at' => '2022-11-09 14:37:00',
+            'updated_at' => '2022-11-09 14:37:00',
+        ])->fresh(); // Use fresh() to load even the null fields.
+        // The target project.
+        $project = ProjectTest::create();
+
+        $oldImage = ImageTest::create(['volume_id' => $volume->id])->fresh();
+        $oldAnnotation = ImageAnnotationTest::create(['image_id' => $oldImage->id]);
+        $oldAnnotationLabel = ImageAnnotationLabelTest::create(['annotation_id' => $oldAnnotation->id]);
+
+        $this->beAdmin();
+        $project->addUserId($this->admin()->id, Role::adminId());
+
+        $response = $this->post("/api/v1/volumes/{$volume->id}/clone-to/{$project->id}");
+        $response->assertStatus(302);
+        $copy = $response->getSession()->get('copy');
+        $newAnnotation = $copy->images()->first()->annotations()->first();
+        $newAnnotationLabel = $newAnnotation->labels()->first();
+
+        $this->assertNotNull($newAnnotationLabel);
+        $this->assertNotEquals($newAnnotationLabel->id, $oldAnnotationLabel->id);
+        $this->assertEquals($newAnnotationLabel->annotation_id, $newAnnotation->id);
+
+        $ignore = ['id', 'annotation_id'];
+        $this->assertEquals(
+            $newAnnotationLabel->makeHidden($ignore)->toArray(),
+            $oldAnnotationLabel->makeHidden($ignore)->toArray()
+        );
+    }
+
+    public function testCloneVolumeVideoAnnotationLabels()
```

Comment:
You could merge this with `testCloneVolumeVideoAnnotations()` as this does mostly the same thing with additional assertions.